app/controllers/search_controller.rb

```diff
@@ -0,0 +1,10 @@
+class SearchController < ApplicationController
+
+  def index
+    @events = EventfulEvent.by_zipcode(token, params[:q])
+  end
+
+  def token
```

Comment:
does this belong in the controller? if we have access to it everywhere, why not keep it in the service or wherever we directly need it?